### ISSUE TITLE
Helm install error - issue with statefulset.yaml

### ISSUE THREAD

**Issue by dstrest** [OctopusSamples/OctopusHelmChart#10]
Hi all,
I'm having an issue running the helm install of Octopus Deploy as written in the readme here https://github.com/OctopusSamples/OctopusHelmChart) 

Executing the following: 

```
helm install octopus octopus/octopusdeploy --set octopus.username=admin --set octopus.password=Password01! --set octopus.acceptEula=Y --set mssql-linux.acceptEula.value=Y --set octopus.masterKey=Px1FZp2ZP4KJHxvIoDsI5A== --namespace octopus-deploy
```
(Password and Master key have been changed)

Results in the following error:
 
`Error: template: octopusdeploy/templates/statefulset.yaml:5:51: executing "octopusdeploy/templates/statefulset.yaml" at <b64enc>: invalid value; expected string`

For your information:
- helm version
`version.BuildInfo{Version:"v3.6.3", GitCommit:"d506314abfb5d21419df8c7e7e68012379db2354", GitTreeState:"clean", GoVersion:"go1.16.5"}`

- kubectl version
```
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.0", GitCommit:"c2b5237ccd9c0f1d600d3072634ca66cefdf272f", GitTreeState:"clean", BuildDate:"2021-08-04T17:56:19Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.20-eks-8c579e", GitCommit:"8c579edfc914f013ff48b2a2b2c1308fdcacc53f", GitTreeState:"clean", BuildDate:"2021-07-31T01:34:13Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
WARNING: version difference between client (1.22) and server (1.18) exceeds the supported minor version skew of +/-1
```

I'm unfortunately a Kubernetes noob, and can't seem to determine where, why, or even if the statefulset.yaml is crapping out, or if it's just me.

Any help would be greatly appreciated.

**Comment by spyder007** [comment]
I believe I had the same issues (#9) but cannot quite tell based on the information above.  I created a PR (#8) that seems to fix the issue for me, perhaps you can try pulling that PR and installing locally.

This repository has not gotten much attention in the last few months.. If I make additional changes they may just live on my own fork of this repository. https://github.com/spyder007/OctopusHelmChart

**Comment by harrisonmeister** [comment]
I believe this has been addressed by #8 in [0.1.36](https://github.com/OctopusSamples/OctopusHelmChart/releases/tag/0.1.36)
